assets/features/podcast/podcastDetailController.js

```diff
@@ -139,6 +139,29 @@ angular.module('Bethel.podcast')
   $scope.$watch('podcastTags', function(newValue) {
     if (angular.isUndefined(newValue)) return;
     $scope.podcast.tags = newValue.split(/ ?, ?/);
-  })
+  });
+
+  $scope.viewPodcastUrl = function($event) {
+    var parentEl = angular.element(document.querySelector('.main-content'));
+
+    $mdDialog.show({
+      clickOutsideToClose: true,
+      parent: parentEl,
+      targetEvent: $event,
+      templateUrl: 'features/podcast/podcastPublishView.html',
+      locals: {feed: $scope.feed, unpublished: $scope.isDemo},
+      controller: PublishPodcastController
+    });
+
+    function PublishPodcastController($scope, $mdDialog, feed, unpublished) {
```

Comment:
I agree with your feedback especially these things:
- I don't like the menu opening directly under the clicked menu button either, for some reason that's the default material menu behavior. Docs say you're supposed to be able to pass an offset y attribute, but it doesn't seem to work in docs or app, so I'll keep poking around there.
- It does feel weird with only 1 menu element right now, so I agree adding upload link from there is a good idea.
- I'll make URL text changes throughout, thanks for that feedback
- I think this is where we'll stick 'Delete Podcast' link when that functionality is ready as well, agree?

Also I see what you're saying about the controller being array convention, I'll make that change and try it again.